pkg/vars/vars.go

```diff
@@ -1,5 +1,41 @@
 package vars
 
-// GatewayClassControllerName is the unique identifier indicating controller
-// responsible for relevant resources.
-const GatewayClassControllerName = "konghq.com/blixt"
+// -----------------------------------------------------------------------------
+// ControlPlane Constants
+// -----------------------------------------------------------------------------
+
+const (
+	// GatewayClassControllerName is the unique identifier indicating controller
+	// responsible for relevant resources.
+	GatewayClassControllerName = "konghq.com/blixt"
+)
+
+// -----------------------------------------------------------------------------
+// Component Defaults
+// -----------------------------------------------------------------------------
+
+const (
+	// DefaultControlPlaneDeploymentName is the name that will be used for the
+	// controlplane's Deployment (by default).
+	DefaultControlPlaneDeploymentName = "blixt-controller-manager"
+
+	// DefaultDataPlaneDaemonSetName is the name that will be used for the
+	// dataplane's DaemonSet (by default).
+	DefaultDataPlaneDaemonSetName = "blixt-dataplane"
+
+	// DefaultNamespace is the namespace used for the controlplane and
+	// dataplane components (by default).
+	DefaultNamespace = "blixt-system"
+
+	// DefaultDataPlaneAPIPort is the network port that will be used to
+	// communicate with the DataPlane API (by default).
+	DefaultDataPlaneAPIPort = 9874
+
+	// DefaultDataPlaneAppLabel indicates the label value that can be used
```

Comment:
/the label value/ -> /the label key/ ? 

Or am I miss-understanding

Reply:
No, it is indeed the label value.